Kempe/Kempe.txt

```diff
@@ -0,0 +1,145 @@
+
+This code is contributed by Riyazul555
+
+
+Time Complexity = O(N^2)
+Space Complexity = O(N)
+
+
+This is the PSEUDOCODE of Kempe's Algrithm 
+
+
+
+
+procedure BFS(nod, G, viz)
+    Q = Queue()
+    viz[nod] = true
+    Q.push(nod)
+    while Q is not empty
+        nod = Q.front()
+        Q.pop()
+        for i in G[nod]
+            if not viz[i]
+                viz[i] = true
+                Q.push(i)
+
+procedure isConnected(n, m, G, vert)
+    viz = array of size nMax initialized to false
+    BFS(vert[0], G, viz)
+    for i in vert
+        if not viz[i]
+            return false
+    return true
+
+procedure isComplete(n, G, vert)
+    for i in vert
+        if size of G[i] is not n - 1
+            return false
+    return true
+
+procedure chromaticPolynomial(k, n, m, G, vert)
+    if m is 0
+        p = 1
+        for i from 1 to n
```

Comment:
There is no style guide for pseudocode indeed (as we do not have a strict definition of the syntax). However, in other algorithm implementations, we used [this syntax](https://github.com/iuliagroza/Algorithms/blob/1f516a04fa02c2c5c7277e2f3b350994cca70386/Floyd-Warshall/Floyd-Warshall.txt#L12) for for-loops. Please replace where applicable.